spec/shc_vaccination/shc_vaccination_must_support_spec.rb

```diff
@@ -0,0 +1,123 @@
+RSpec.describe SHCVaccinationTestKit::MustSupportTest do
+  let(:suite) { Inferno::Repositories::TestSuites.new.find('shc_vaccination') }
+  let(:session_data_repo) { Inferno::Repositories::SessionData.new }
+  let(:runner) { Inferno::TestRunner.new(test_session: test_session, test_run: test_run) }
+  let(:test_session) do
+    Inferno::Repositories::TestSessions.new.create(test_suite_id: suite.id)
+  end
+  let(:request_repo) { Inferno::Repositories::Requests.new }
+  let(:group) { suite.groups.first.groups.find { |g| g.id.include?('shc_vaccination-Group01-shc_must_support_group')} }
+
+  def run(runnable, inputs = {})
```

Comment:
If we merge this we need to factor out the methods that are now auto-injected from inferno core. See [inferno_core/spec/runnable_context.rb](https://github.com/inferno-framework/inferno-core/blob/main/spec/runnable_context.rb)